packages/arb-token-bridge-ui/.e2e.env.sample

```diff
@@ -1,3 +1,4 @@
-NETWORK_NAME="goerli"
-SECRET_WORDS="your, seed, phrase, for, your, e2e, test, metamask, extension, in, comma, separated, format"
-ADDRESS="0xyour_address_here"
+NETWORK_NAME=localhost
+PRIVATE_KEY_LOCAL=b6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659
```

Comment:
We don't use `PRIVATE_KEY` because Synpress is looking for it to set up MetaMask and we use a new wallet for that